test/Combinatorics/Matroids/realization_space.jl

```diff
@@ -0,0 +1,118 @@
+@testset "matroid realization spaces" begin
+
+
+    M1 = fano_matroid()
+    M2 = pappus_matroid()
+    M3 = vamos_matroid()
+    M4 = matroid_from_bases([['a','b'], ['a','c'], ['a','d'], ['b','c'], ['b','d']], ['a','b','c','d']))
+    
+    R1, mat_M1 = Oscar.realization_space_matrix(M1, [1,2,4], GF(2));
+    R2, mat_M2 = Oscar.realization_space_matrix(M2, [2,3,8], QQ);
+    R3, mat_M3 = Oscar.realization_space_matrix(M3, [2,4,5,7], ZZ);
+    
+    x1 = gens(R1)
+    x2 = gens(R2)
+    x3 = gens(R3)
+      
+    @testset "realization_space_matrix" begin
+        
+        @test coefficient_ring(R1) == GF(2)
+        @test coefficient_ring(R2) == QQ
+        @test coefficient_ring(R3) == ZZ
+        
+        @test length(x1) == 3
+        @test length(x2) == 8
+        @test length(x3) == 9
+	
+        @test mat_M1 ==  matrix(R1, [1 0 1 0 1 0 1; 0 1 1 0 0 1 x1[2]; 0 0 0 1 1 x1[1] x1[3]])
+	@test mat_M2 ==  matrix(R2, [1 1 0 0 1 1 1 0 1; 1 0 1 1 x2[1] x2[3] x2[5] 0 x2[7]; 0 0 0 1 x2[2] x2[4] x2[6] 1 x2[8]])
```

Comment:
Pease don't mix tabs and spaces. For OSCAR: just don't use tabs.

Reply:
Tabs removed, think I got them all.